content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,37 @@
+
+Finally, in my fourth semester, I got to meet and be a part of this awesome clan of FOSS patrons: ALiAS.
+
+## It all started with a commit ...
+January 8, 2017.
+
+I was just lying in my room going through all my FB, Instagram and WhatsApp updates when I saw an invite link to the then-called 'ASET ALiAS group' (the first WhatsApp group of ALiAS). At first, I discarded the message as I thought of it to be another entrant in the list of not-so-enthusiastic tech clubs. But later that day, I casually went through its website and found something that lacked in other clubs - a GitHub repo of their website with well-defined contribution guidelines. So that was about some serious start.
+
+I went through the repo and found a layout bug in a webpage's CSS code. Thought of it as a nice & small start, and I did the usual - fork, clone, fix, commit, and push. That single, tiny commit landed me into the first batch of ALiAS members and their first ever group meet. Little I knew, how a single commit was gonna change the course of my next two years of college.
+
+## ... And behold the community!!
+Starting from weekly meetups to some in-house workshops, ALiAS opened up the gates for all of us and people coming after, towards some of the biggest FOSS communities in Delhi-NCR like PyDelhi, ILUG-D, etc. We called people from those communities to our campus for talks, workshops, etc. We came up with our projects for the club newbies to get their hands dirty on, and also tried to organize a hackathon (which didn't go well though). But all that proved to be a large addition to our fun and learning.
+
+And if that wasn't enough, getting along with the fellow ALiASians to various FOSS meetups, devsprints, hackathons was another level of experience. Victory for us wasn't about winning in the events, but leaving our impression on people around us. And I can say, we did win in that case.
+
+So going from a small club revived after years, to a huge community of people like me, the end of which can't (and shouldn't) be defined, ALiAS helped me to bring a new spark in my life. And I can say that no one or nothing can put out this spark, ever.
+
+## Part of the journey is the end, but not for us ....... not us
+Speaking in the present time, my college life had reached its end. All those moments in four years are now just memories. As the universal truth is, that everything comes to an end somewhere at some time. But if you ask the same for my relationship with ALiAS, I'll say, "Not yet, we're still on."
+
+ALiAS has grown beyond a club or a community. It's now more like a part of us. It's inseparable from us. As long as we go on, it goes on with us. This might not be true for everyone else, but it is for me. And I say this as an active member and contributor of Amity Linux Assistance Sapience a.k.a. ALiAS :P.
+
+
```

Comment:
A conclusion is missing, what we like to know is definitive things you got from ALiAS, where are the pictures? 
This could be the Paytm hack you participated in, talks, meetups with ILuGD. This is good article, help us make this great.